components/TwoPlayerGrid/index.js

```diff
@@ -5,9 +5,13 @@ import MemoryGrid from "../MemoryGrid";
 export default function SinglePlayerGrid() {
   const setPlayer = store((state) => state.setPlayer);
   const player = store((state) => state.player);
+  const setMatches = store((state) => state.setMatches);
 
   const { handleReveal, handleConceal, shuffledImages, compareImages } =
     useMemoryGame({
+      onMatchSolved: () => {
```

Comment:
I think we could shorten this a little to the following. 

``` js
onMatchSolved: () => {
  setMatches();
},
```

to

`onMatchSolved: setMatches,`